@here/harp-mapview/lib/MapView.ts

```diff
@@ -1490,7 +1490,28 @@ export class MapView extends THREE.EventDispatcher {
     }
 
     /**
-     * Moves the camera to specified geocoordinates, sets desired zoom level and adjusts yaw and
+     * The method that sets the camera to the desired angle (`pitch`) and distance (in meters) to
+     * the `target` location.
+     *
+     * @param target The location to look at.
+     * @param distance The distance of the camera to the target in meters.
+     * @param pitch The camera tilt angle in degrees.
+     * @param yaw The camera yaw angle in degrees.
```

Comment:
We should align with harp-native, i.e. use azimuth and tilt

Reply:
You mean, replace `yaw` and `pitch` with `azimuth` and `tilt`? We use these values extensively and they are exposed externally. 

Should I do this in this ticket?

Reply:
Another thing: I chose the parameter `distance` in order to align with native as well, but the competition prefers a zoom level number. With `setCameraGeoLocationAndZoom` that is also what we did. Do we switch to distance in meters indeed? Setting a global view might be painful.

Reply:
If we put zoomlevel here instead of distance, this means we have to define our zoomlevel as distance to the look-at point. Currently this is only true if the tilt is less than 60 degrees and we have a flat map (i.e. no globe and no terrain). 
In mapview Utils.ts we have `calculateDistanceToGroundFromZoomLevel` but that also makes not much sense b/c it is not the distance to ground.